cms/bundles/tests/test_viewsets.py

```diff
@@ -183,6 +185,26 @@ def test_add_view_passes_access_token_to_form(self):
 
 
 class BundleViewSetEditTestCase(BundleViewSetTestCaseBase):
+    @staticmethod
+    def chooser_panel_display(page) -> str:
+        return f"{page.title} ({page.get_status_display()}, {page.release_date_value})"
+
+    def _assign_release_calendar_page_to_bundle(
+        self,
+        title: str = "Future Release Calendar page",
+        status: ReleaseStatus = ReleaseStatus.PROVISIONAL,
```

Comment:
You don't seem to use these defaults anyway, so not sure why you have them. It doesn't make sense in this case.